File: src/modelo/acordes.ts
// src/cancion.ts
export class Parte {
    nombre: string;
    acordes: string[];

    constructor(nombre: string, acordes: string[]) {
        this.nombre = nombre;
        this.acordes = acordes;
    }
}


export class Acordes {
    partes: Parte[];
    orden_partes: number[];

    constructor(partes: Parte[], orden_partes: number[]) {
        this.partes = partes;
        this.orden_partes = orden_partes;
    }
     
}

File: construir_datos/acordes.py
class Parte:
    def __init__(self, nombre: str, acordes: list):
        self.nombre = nombre
        self.acordes = acordes

class Acordes:
    def __init__(self, partes: list, orden_partes: list):
        self.partes = partes
        self.orden_partes = orden_partes

    def toJson(self):
        return {
            'partes': [parte.__dict__ for parte in self.partes],
            'orden_partes': self.orden_partes
        }